soc/arm/silabs_exx32/common/soc_power.c

```diff
@@ -12,15 +12,15 @@ LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
 
 /*
  * Power state map:
- * POWER_STATE_SLEEP_1: EM1 Sleep
- * POWER_STATE_SLEEP_2: EM2 Deep Sleep
- * POWER_STATE_SLEEP_3: EM3 Stop
+ * PM_STATE_RUNTIME_IDLE: EM1 Sleep
+ * PM_STATE_SUSPEND_TO_IDLE: EM2 Deep Sleep
+ * PM_STATE_STANDBY: EM3 Stop
```

Comment:
This doesn't seem to match https://github.com/zephyrproject-rtos/zephyr/issues/31162#issuecomment-757337571, but I guess that can be fixed later.

Reply:
The comment was not super conclusive and was not clear about EM2 and EM3, also right now is not clear how to distinguish between them, so just keep this way for clarity sake.